### PR TITLE
[TMVA][SOFIE] Broadcast bias of Conv and ConvTranspose operators

### PR DESCRIPTION
**Pull request by axmat**
# This Pull request:
Broadcast the bias of a Conv or ConvTranspose operator when the input image is batched and there's no Session class.

- [x] tested changes locally

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default, ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On

**Review by lmoneta — APPROVED** [review]
LGTM!
Thank you Ahmat for this improvement. I think it is good to have such function that can be used in the generate the code for broadcasting the bias tensor

File: tmva/sofie/inc/TMVA/SOFIE_common.hxx [hunk 1]
```diff
@@ -108,6 +108,46 @@ T* Unidirectional_broadcast(const T* original_data, const std::vector<size_t> or
 std::vector<size_t> Multidirectional_broadcast(std::vector<size_t> input1_shape, std::vector<size_t> input2_shape);
 std::string Clean_name(std::string input_tensor_name);
 
+template<typename T>
+T* BroadcastConvBias(const T* data, const size_t channel, const std::vector<size_t>& targetShape) {
+   size_t size = targetShape.size();
+   size_t index = 1;
+   if (targetShape[1] != channel) {
+      std::stringstream ss;
```
Comment (vepadulano) [inline tmva/sofie/inc/TMVA/SOFIE_common.hxx:116]: This is missing an include right?

Comment (lmoneta) [reply tmva/sofie/inc/TMVA/SOFIE_common.hxx:116]: Yes, I think so, thanks for finding this out. 
@axmat , can you please add an `#include <sstream>` ?
You can maybe add in the ConVTranspose PR

**Pull request merged**